### PR TITLE
React demo

### PR DESCRIPTION
**Pull request by SuduguTejasvi**
React myntra card Demo application

### CODE REVIEW

**Comment by SuduguTejasvi** [comment]
image

**Review by SaiSree-0517 — COMMENTED** [review]
can you just brief where have you added font family

File: src/Themes/index.tsx
Comment (SaiSree-0517) [inline src/Themes/index.tsx:1]: where have you used this theme file

Comment (SuduguTejasvi) [reply src/Themes/index.tsx:1]: I downloaded a font from Google and pasted it into the fonts folder within the public directory. I declared the font using @font-face in styles.css located in the src directory, and then imported styles.css in index.tsx. I used that font in themes.

Comment (SuduguTejasvi) [reply src/Themes/index.tsx:1]: I used a theme file in App.tsx. I imported the file and provided the themes in ThemeProvider

**Comment by SaiSree-0517** [comment]
check the styling User experience designer font weight is not matching the figma

File: src/components/organism/Card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+export default function MyntraCard() {
+  return (
+    <Grid container spacing={4}>
+      <Grid item xs={2} container justifyContent="flex-end" style={gridItemStyle}>
+        <img src="/asserts/images/myntra.svg" style={{ height: '45px', width: '45px' }} alt="Myntra Logo" />
+      </Grid>
+      <Grid item xs={10}>
+        <Grid container spacing={2}>
+          <Grid item xs={12}>
+            <Title v1="h3" v2="h4" v3="h6" path="/asserts/images/more.svg" text1="User Experience Designer" text2="Myntra" text3="Hitech city Hyderabad -500072" />
```
Comment (SaiSree-0517) [inline src/components/organism/Card/index.tsx:18]: get these values from constants file

**Review by SaiSree-0517 — CHANGES_REQUESTED** [review]

File: src/components/organism/Card/index.tsx (hunk 1 above)
Comment (SuduguTejasvi) [reply src/components/organism/Card/index.tsx]: Mam, I added a constants file in src and imported images using these constants. Could you please review it and let me know if any changes are required? I also adjusted the font weight according to the Figma design.

image

**Review by SaiSree-0517 — APPROVED** [review]

**Pull request merged**